src/components/Transaction.js

```diff
@@ -1,14 +1,18 @@
-import React from 'react'
+import React, {Component} from 'react'
+
+export default class Transaction extends Component {
```

Comment:
again, this can be a functional component. I would avoid changing this to a class unless you need to.